src/apiCalls.js

```diff
@@ -15,4 +15,15 @@ export const updateSettings = (settings) => {
 
 export const getRandomContent = (duration, category) => {
   return axios.get(`${rootUrl}/rand_exercise?duration=${duration}&category=${category}`)
+}
+
+export const postSession = (session) => {
+  console.log(session)
```

Comment:
Do we wanna remove this `console.log()` here? I'm happy to do that on my next pass through 😊